app/domain/authentication/authn_jwt/identity_providers/create_identity_provider.rb

```diff
@@ -61,11 +62,24 @@ def validate_identity_configuration
         end
 
         def multiple_identities_configured
-          identity_from_decoded_token_provider.identity_available? && identity_from_url_provider.identity_available?
+          token_app_property_is_available? && identity_from_url_available?
         end
 
         def no_identities_configured
-          !identity_from_decoded_token_provider.identity_available? && !identity_from_url_provider.identity_available?
+          !token_app_property_is_available? && !identity_from_url_available?
+        end
+
+        def token_app_property_is_available?
+          @identity_from_decoded_token_available ||= @check_authenticator_secret_exists.call(
+            conjur_account: @authentication_parameters.account,
+            authenticator_name: @authentication_parameters.authenticator_name,
+            service_id: @authentication_parameters.service_id,
+            var_name: TOKEN_APP_PROPERTY_VARIABLE
+          )
+        end
+
+        def identity_from_url_available?
```

Comment:
how about this?
```suggestion
        def identity_from_url?
```

Reply:
But this is not the identity_from_url itself but just indication it is available

Reply:
if the method name was `identity_from_url` then the reader would see it as the value itself. Having it as `identity_from_url?` indicates that it's a bool value where `true` is that it is not nil and `false` is that it is nil. I think that it is the same as `identity_from_url_available?` and more readable and ruby-style